regression/osd_regression/main.cpp

```diff
@@ -639,17 +639,17 @@ main(int argc, char ** argv) {
     }
 
     int width=10, height=10;
-    
+
     static const char windowTitle[] = "OpenSubdiv OSD regression";
     if (! (g_window=glfwCreateWindow(width, height, windowTitle, NULL, NULL))) {
         printf("Failed to open window.\n");
         glfwTerminate();
         return 1;
     }
     glfwMakeContextCurrent(g_window);
-    
+
 #if defined(OSD_USES_GLEW)
-    if (GLenum r = glewInit() != GLEW_OK) {
+    if (GLenum r = (glewInit() != GLEW_OK)) {
```

Comment:
Adding parenthesis here shouldn't have any effect as "=" is one of the lowest precedence operators.  Both the old and new code are flawed in that the RHS of the assignment evaluates to a boolean value and is then assigned to "GLenum r".  The return code from glewInit() is therefore lost when it is most needed in the error message that follows -- it will always print "error = 1".

Incidentally, it appears this test of the result of glewInit() is coded the same way in all GL examples.